src/main/java/io/dockstore/provision/S3CmdPlugin.java

```diff
@@ -150,6 +152,17 @@ private void setConfigAndClient() {
          */
         public boolean uploadTo(String destPath, Path sourceFile, Optional<String> metadata) {
             setConfigAndClient();
+            String modifedChunkSize = "";
+            try {
+                long sizeInBytes = Files.size(sourceFile);
+                long sizeInMegabytes = sizeInBytes/1000000;
+                if (sizeInMegabytes > DEFAULT_CHUNK_SIZE*MAX_PARTS) {
```

Comment:
I may be missing it, but is there any cap on chunk size? It looks like newChunkSize can grow past the 5GB limit.

Reply:
The other question for documentation. This seems to make as many chunks as possible, is there any benefit to this?

Reply:
I let s3cmd handle the 5 GB limit in the event that a new s3cmd client suddenly raises the cap.  s3cmd reports "ERROR: Parameter problem: Chunk size 6000 MB is too large, must be <= 5120 MB. Please adjust --multipart-chunk-size-mb"

Reply:
As for as many chunks as possible, s3cmd lets you resume the download or upload if it fails halfway.  Having more chunks is safer since the file is so large.  Considering s3cmd set the 15 MB as the default size even though 5 GB is allowed, I think it's alright that we stay close to 15 MB.

Reply:
Ok, sounds good. Please document both decisions